typed-racket-lib/typed-racket/private/type-contract.rkt

```diff
@@ -399,8 +407,13 @@
         (if (from-typed? typed-side)
             (and/sc sc any-wrap/sc)
             sc))
+      ;(eprintf "predef: ~s ~s ~s\n" predef-contracts type typed-side)
       (cached-match
```

Comment:
the thought is we'll want to debug this fequently and so we're leaving in the commented our `eprintf`s, is that right?